### PR TITLE
Styling fullscreen media

### PR DESCRIPTION
**Pull request by zufrieden**

### CODE REVIEW

**Comment by zufrieden** [comment]
@Yago review welcome ;-)

File: assets/components/atoms/overlay.hbs [hunk 1]
```diff
@@ -3,8 +3,17 @@ notes: |
...
-  {{> alert alert-variant="success" }}
+<div id="overlay" class="overlay">
+  {{#if fullscreen}}
+  <div class="fullscreen">
+    <a href="#" class="btn-close" >
+      <span class="cross">×</span>
+    </a>
+    <img src="http://notrehistoire.ch.s3.amazonaws.com/photos/2015/03/d4ed767309c114f4.jpeg" alt="Salavaux ( Tea-Room Friederich)" class="img-responsive" style="position:absolute;top:0;left:50%;width:1611.418320610687px;height:1321px;margin-top:0;margin-left:-805.7091603053435px;">
+  </div>
+  {{^}}
```
Comment (Yago) [inline assets/components/atoms/overlay.hbs]: No need of empty “else”

File: assets/sass/atoms/_overlays.scss [hunk 2]
```diff
@@ -11,4 +11,24 @@
     right: 0px;
     background: rgba(255, 255, 255, 0.8);
   }
+  .fullscreen{
+    z-index: 100000;
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 100%;
+    height: 100%;
```
Comment (Yago) [inline assets/sass/atoms/_overlays.scss]: seems dangerous to me... Even if it's a kind of “hack”, use `bottom: 0px;`, it's saffer. We can also use `height: 100vh;` as cleaner way to do it (Switzerland support -> 95.14%) :wink:

File: assets/sass/atoms/_overlays.scss [hunk 3]
```diff
@@ -11,4 +11,24 @@
...
+    .btn-close{
+      z-index: 100001;
+      position: absolute;
+      top: 0px;
+      background: rgb(255, 0, 0) none repeat scroll 0% 0%;
+      color: rgb(255, 255, 255);
+      text-decoration: none;
+      font-size: 1.4em;
+      padding: 3px 14px;
+      right: 0px;
```
Comment (Yago) [inline assets/sass/atoms/_overlays.scss]: It's nicer to group all similar property. So here, `right` with `top` and `position`

File: assets/templates/detail-fullscreen.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{{> navbar }}
+
+{{> panel-left }}
+{{> panel-right }}
+{{> overlay fullscreen="true" }}
```
Comment (Yago) [inline assets/templates/detail-fullscreen.html]: For boolean option, you can pass a boolean parameter :wink:  -> `{{> overlay fullscreen=true }}`

File: assets/components/atoms/overlay.hbs [hunk 5]
```diff
@@ -3,8 +3,17 @@ notes: |
...
+  - **fullscreen** - String ('true')
 ---
-<div id="overlay" class="overlay"></div>
-<div style="display:none;" id="js-alert-zone">
-  {{> alert alert-variant="success" }}
+<div id="overlay" class="overlay">
+  {{#if fullscreen}}
+  <div class="fullscreen">
+    <a href="#" class="btn-close" >
+      <span class="cross">×</span>
```
Comment (Yago) [inline assets/components/atoms/overlay.hbs]: No need of span.cross, and `&times;`is nicer

**Pull request merged**